Assets/Scripts/DialsManager.cs

```diff
@@ -957,6 +1010,18 @@ void DeactivateUnavailableDials(GameObject countryDialBoard, PlaneDataFromName.P
         {
             allOilPressureTemps[i].SetActive(false);
         }
+
+        GameObject[] allOilComboTempsArray = GameObject.FindGameObjectsWithTag("oil temp combo");
+        List<GameObject> allOilComboTemps = new List<GameObject>();
+        allOilComboTemps.AddRange(allOilComboTempsArray);
+
+        foreach (GameObject oilTemp in oilTempComboObjects)
+            allOilComboTemps.Remove(oilTemp);
+
+        for (int i = 0; i < allOilComboTemps.Count; i++)
+        {
+            allOilComboTemps[i].SetActive(false);
+        }
```

Comment:
If not using brackets for one line loops (as per line `1019`, keep it consistent